### ISSUE TITLE
[toc2] feature requests:   reordering notebook sections  by draging  toc entries

### ISSUE THREAD

**Issue by chiwhalee** [ipython-contrib/jupyter_contrib_nbextensions#1302]
Currently, moving multiple cells is supported by jupyter notebook, through select/cut/paste operations. It is clearly more efficient, if one could move a group of cells within a section, by just draging the  corresponding toc entries.  I think this may make the toc  plugin more powerful, thanks.

**Comment by nklein16** [comment]
Yes, I would also love to see this implemented!  I was going to suggest this myself when I found your suggestion already posted.  TOC2 has already made a tremendous difference for me in how I interact with jupyter notebooks.  Having said that, moving cells around is a regular issue for me.  If this were to be implemented it would really make a tremendous difference in my productivity!
